jaxkern/approximations/rff.py

```diff
@@ -0,0 +1,35 @@
+from ..base import AbstractKernel
+from ..computations import BasisFunctionComputation
+from jax.random import KeyArray
+from typing import Dict, Any
+
+
+class RFF(AbstractKernel):
+    def __init__(self, base_kernel: AbstractKernel, num_basis_fns: int) -> None:
+        self._check_valid_base_kernel(base_kernel)
```

Comment:
Are we deliberately light on doc strings?

Reply:
No - sloppy PR. Resolved.